tests/acceptance/shared/util/constants.go

```diff
@@ -0,0 +1,82 @@
+
+import (
+	"flag"
+
+	f "github.com/k3s-io/k3s/tests/acceptance/core/service/flag"
+)
+
+// global configurations and flag vars
+var (
+	Destroy = flag.Bool("destroy", false, "a bool")
+	Arch    = flag.String("arch", "amd64", "a string")
+
+	InstallType               = &f.InstallTypeValue{}
+	ExpectedValueUpgradedHost string
+	ExpectedValueUpgradedNode string
+	CmdHost                   string
+	ExpectedValueHost         string
+	CmdNode                   string
+	ExpectedValueNode         string
+	Description               string
+	InstallUpgradeFlag        f.MultiValueFlag
+	TestCase                  f.TestConfigFlag
+
+	KubeConfigFile   string
+	ServerIPs        string
+	AgentIPs         string
+	NumServers       int
+	NumAgents        int
+	AwsUser          string
+	AccessKey        string
+	RenderedTemplate string
+	ExternalDb       string
+	ClusterType      string
+)
+
+// global cmd vars
+var (
+	GetAll                    = "kubectl get all -A --kubeconfig="
+	GetPodsWide               = "kubectl get pods -o wide --no-headers -A --kubeconfig="
+	GetNodesWide              = "kubectl get nodes --no-headers -o wide -A --kubeconfig="
+	GetAppLoadBalancer        = "kubectl get pods -o=name -l k8s-app=nginx-app-loadbalancer --field-selector=status.phase=Running --kubeconfig="
+	GetIngress                = "kubectl get ingress -o jsonpath='{.items[0].status.loadBalancer.ingress[*].ip}' --kubeconfig="
+	GetIngressRunning         = "kubectl get pods  -l app=ingress -l k8s-app=nginx-app-loadbalancer --field-selector=status.phase=Running  --kubeconfig="
+	GetImageLocalPath         = "kubectl describe pod -n kube-system local-path-provisioner- "
+	GetExternalNodeIp         = "kubectl get node --output=jsonpath='{range .items[*]} {.status.addresses[?(@.type==\"ExternalIP\")].address}' --kubeconfig="
+	GetLoadbalancerSVC        = "kubectl get service nginx-loadbalancer-svc --output jsonpath={.spec.ports[0].port} --kubeconfig="
+	GetPodDnsUtils            = "kubectl get pods dnsutils --kubeconfig="
+	ExecDnsUtils              = "kubectl exec -t dnsutils --kubeconfig="
+	GetPodVolumeTestRunning   = "kubectl get pods -l app=volume-test --field-selector=status.phase=Running --kubeconfig="
+	GetNodeport               = "kubectl get pods -l k8s-app=nginx-app-nodeport --field-selector=status.phase=Running --kubeconfig="
+	GetClusterIp              = "kubectl get pods -l k8s-app=nginx-app-clusterip --field-selector=status.phase=Running --kubeconfig="
+	GetPodTestWithAnnotations = "kubectl get pod test-pod -o yaml --kubeconfig="
+	DeletePod                 = "kubectl delete pod -l app=volume-test --kubeconfig="
+)
+
+// constant names, asserts and cmds
+const (
+	RestartK3s        = "sudo systemctl restart k3s*"
+	TestClusterip     = "test-clusterip"
+	NginxClusterIpSVC = "nginx-clusterip-svc"
+	NginxNodePortSVC  = "nginx-nodeport-svc"
+	TestNodePort      = "test-nodeport"
+	TestDaemonset     = "test-daemonset"
+	TestIngress       = "test-ingress"
```

Comment:
This constant is only ever used in the `core/testcase/ingressdns.go` file. We should move it to that package. The same goes for many of the constants in this file. If we are only using them in a single location, we should nove have them exported in a shared package.